anesthetic/samples.py

```diff
@@ -389,39 +386,95 @@ def importance_sample(self, logL_new, action='add', inplace=False):
 
         return modify_inplace(self, samples, inplace)
 
-    def _limits(self, paramname):
-        limits = self.limits.get(paramname, (None, None))
-        if limits[0] == limits[1]:
-            limits = (None, None)
-        return limits
 
-    def _reload_data(self):
-        self.__init__(root=self.root)
-        return self
+class MCMCSamples(Samples):
+    """Storage and plotting tools for MCMC samples.
 
-    def copy(self, deep=True):
-        """Copy which also includes mutable metadata."""
-        new = super().copy(deep)
-        if deep:
-            new.tex = copy.deepcopy(self.tex)
-            new.limits = copy.deepcopy(self.limits)
-        return new
+    Extends the pandas.DataFrame by providing plotting methods and
+    standardising sample storage.
 
-    _metadata = WeightedDataFrame._metadata + ['tex', 'limits',
-                                               'root', 'label']
+    Example plotting commands include
+        - ``mcmc.plot_1d(['paramA', 'paramB'])``
+        - ``mcmc.plot_2d(['paramA', 'paramB'])``
+        - ``mcmc.plot_2d([['paramA', 'paramB'], ['paramC', 'paramD']])``
```

Comment:
We should update this docstring so that it makes clearer what MCMCSamples adds

Reply:
> We should update this docstring so that it makes clearer what MCMCSamples adds

So far it barely adds anything, but that might change in the future.
I've adapted the docstring to match the `NestedSamples` one for now in [69b05dd](https://github.com/williamjameshandley/anesthetic/pull/209/commits/69b05dd529e480feb175e83ebdbaca843d708305).